### DISCUSSION TITLE
How do you split along character ranges

### DISCUSSION THREAD

**Discussion by fschroiff** [fontsource/fontsource#999] [Q&A]
Hi, how does Fontsource split font files into smaller files with certain character ranges? I'd like to be able to do that for my own fonts sometimes. For fonts that are not available on Fontsource, for example. Or for a version of Inter that I have exported with certain opentype features enabled that Fontsource does not have. 

What is the stack that you use to split the fonts? Is this manual or automated work? Is this a well known technique? Is it https://github.com/fonttools/fonttools? Could someone point me to a how-to?

**Comment by n0099** [comment]
It's done by Google Fonts: https://github.com/google/fonts/issues/3870
https://github.com/fontsource/fontsource/discussions/674#discussioncomment-5953379
https://github.com/fontsource/fontsource/issues/949#issuecomment-2048783777
https://github.com/fontsource/fontsource/issues/1075#issuecomment-3301521374
